src/vmware/azext_vmware/_help.py

```diff
@@ -846,15 +846,15 @@
     short-summary: Create a segment by ID in a private cloud workload network.
     examples:
     - name: Create a segment by ID in a workload network.
-      text: az vmware workload-network segment create --resource-group group1 --private-cloud cloud1 --segment segment1 --display-name segment1 --connected-gateway /infra/tier-1s/gateway --revision 1 --dhcp-ranges 40.20.0.0 40.20.0.1 --gateway-address 40.20.20.20/16 --port-name port1
+      text: az vmware workload-network segment create --resource-group group1 --private-cloud cloud1 --segment segment1 --display-name segment1 --connected-gateway /infra/tier-1s/gateway --revision 1 --dhcp-ranges 40.20.0.0 40.20.0.1 --gateway-address 40.20.20.20/16

```

Comment:
May I ask why `--port-name` is removed in the example?

Reply:
Good question. Because the --port-name is read-only property in swagger. And this argument was not work.